API.md

```diff
@@ -414,7 +478,45 @@ Telemetry requests support time bounded queries. A call to a _Telemetry Provider
 }
 ```
 
-#### Telemetry Formats
+In this case, the `domain` is the currently selected time-system, and the start and end dates are valid dates in that time system.
+
+#### Request Strategies **draft**
+
+To improve performance views may request a certain strategy for data reduction.  These are intended to improve visualization performance by reducing the amount of data needed to be sent to the client.  These strategies will be indicated by additional parameters in the request options.  You may choose to handle them or ignore them.  
+
+Note: these strategies are currently being tested in core plugins and may change based on developer feedback.
+
+##### `latest` request strategy
+
+example:
+```javascript
+{
+    start: 1487981997240,
+    end: 1487982897240,
+    domain: 'utc',
+    strategy: 'latest',
+    size: 1
+}
+```
+
+This strategy says "I want the lastest data point in this time range".  A provider which recognizes this request should return only one value-- the latest-- in the requested time range.  Depending on your back-end implementation, performing these queries in bulk can be a large performance increase.  These are generally issued by views that are only capable of displaying a single value and only need to show the latest value.
```

Comment:
The size attribute is redundant in this case, we should get rid of it. Unless there is a case for supporting latest N results, in which case we should clarify this in the text.